src/Database/Statement/BufferedStatement.php

```diff
@@ -337,7 +338,7 @@ public function next()
      *
      * @return \Cake\Database\StatementInterface
      */
-    public function getInnerStatement()
+    public function getInnerStatement(): \Cake\Database\StatementInterface
```

Comment:
Should add "use" statement instead of using FQCN. There are other such cases to in the PR.

Reply:
There are sniffers that can automate this. We have one in spryker namespace for example.

Reply:
I ran a tool, and didn't see this when I was cleaning up. I'll fix this now.